deployments/monitoring/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v2
+name: monitoring
+version: 1.0.0
+description: SQuaRE packaging of monitoring apps
+dependencies:
+  - name: influxdb2
+    version: 2.0.10
```

Comment:
Any reason not to use 2.0.11? I see the chart is published [here](https://github.com/influxdata/helm-charts/releases/tag/influxdb2-2.0.11).

Reply:
Did they actually publish it?  When I tried, it wasn't downloadable.  I think they built it during the time last week when GitHub Actions were misbehaving.

I'll try redeploying dev and see if it works this time.

Reply:
Yup:

```
Getting updates for unmanaged Helm repositories...
...Successfully got an update from the "https://helm.influxdata.com/" chart repository
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "jupyterhub" chart repository
Update Complete. ⎈Happy Helming!⎈
Error: can't get a valid version for repositories influxdb2. Try changing the version constraint in Chart.yaml
```

Reply:
I'm going to merge it.  We will update the version when there's one that is downloadable.